script/fork-helpers/SimulateVote.s.sol

```diff
@@ -109,6 +109,22 @@ contract SimulateVote is Script, DeploymentFixtures, ForkHelpersCommon {
         IBurner burner = IBurner(locator.burner());
 
         vm.startBroadcast(admin);
+
+        accounting.revokeRole(accounting.SET_BOND_CURVE_ROLE(), address(csm));
+        accounting.revokeRole(
+            accounting.SET_BOND_CURVE_ROLE(),
+            accounting.getRoleMember(accounting.SET_BOND_CURVE_ROLE(), 0)
+        );
```

Comment:
Why not an exact address here?

Reply:
to not overcomplicate the script for tests
this address is in the deploy scripts only, not in the configs